packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -212,63 +210,87 @@ function EmbedOptionContainer(
   }
 }
 
-function handleUp(selection: DefaultSelectionManager) {
-  const { state } = selection;
-  if (state.selectedBlocks.length === 1) {
-    const selectedModel = getModelByElement(state.selectedBlocks[0]);
-    if (!matchFlavours(selectedModel, ['affine:divider'])) {
-      return;
+function handleUp(
+  selection: DefaultSelectionManager,
+  signals: DefaultPageSignals,
+  e: KeyboardEvent
+) {
+  const nativeSelection = window.getSelection();
+  if (nativeSelection?.anchorNode) {
+    const model = getStartModelBySelection();
+    const activeContainer = getContainerByModel(model);
+    const activePreNodeModel = getPreviousBlock(activeContainer, model.id);
+    const editableContainer = getBlockElementByModel(model)?.querySelector(
+      '.ql-editor'
+    ) as HTMLElement;
+    const range = nativeSelection.getRangeAt(0);
+    const { height, left, top } = range.getBoundingClientRect();
+
+    // TODO resolve compatible problem
```

Comment:
This looks like a false comment since we don't have the compatibility problem by having `caretRangeFromPoint` to support Firefox.